build-configs/configs/BuildConfigType.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import type { ThemeType } from '../themes/ThemeType'
+import type { FeatureFlagsType } from './featureFlags'
+
+export type BuildConfigType = {|
+  appTitle: string,
+  cmsUrl: string,
+  switchCmsUrl?: string,
```

Comment:
maybe rather call this `testCmsUrl`

Reply:
Not really correct since for the integreat-test-cms config it is the other way round.